_pages/index.md

```diff
@@ -30,7 +30,6 @@ classification for discussion._
 * [Front-End Disciplines]({{site.baseurl}}/frontend)
 * [Third-Party Integrations]({{site.baseurl}}/integrations)
 * [Workflow Best Practices]({{site.baseurl}}/workflow)
-* [Example Workflows]({{site.baseurl}}/example-workflows)
```

Comment:
Curious as to why we are removing this link, we still refer to this page in `workflow.md` so presumably it's still relevant?

Reply:
Since it was linked on the main Workflow page, I thought I might simplify this table of contents, making it just a top-level list by removing it. 

The other approach I considered was to create a more complete table-of-contents, and nest sub-pages as sub-bullets under each main topic.

What do you think? I only had a slight preference for the top-level-only table of contents.

Reply:
Ooh, makes sense. I prefer a simplified TOC myself -- in this case I believe the content isn't so large that users will get lost or confused.